examples/BAPC2014/Armybase.lhs

```diff
@@ -50,10 +50,10 @@ A value of type Half should still be halved. I.e. 'Half 2x = x'
 
 > showValue :: Half -> String
 > showValue (Half i) = concat [ show $ i `div` 2
->                             , if odd i then ".5" else ""
+>                             , if myOdd i then ".5" else ""
 >                             ]
 >   where
->     odd x = x `mod` 2 /= 0
+>     myOdd x = x `mod` 2 /= 0
```

Comment:
I don't recall why I defined a different 'odd' function than the one in the prelude. I think it would suffice just to get rid of the local where clause entirely.

Reply:
probably address in different pr